packages/keyda-demo/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "keyda-demo",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.1"
+  },
```

Comment:
The keyda-demo package is basically made with `create-react-app`. 
It could be enhanced to more good-looking. And also deployed to the public.